src/templates/app/user/authenticate.jsx

```diff
@@ -41,7 +41,20 @@ const Authenticate = () => (
             <p>{it.L('We will inform you when your account needs to be authenticated.')}</p>
         </div>
 
-        <div id='authentication_tab' className='gr-padding-20'>
+        <div id='residence_selection' className='center-text gr-padding-20'>
+            <h1 className='gr-padding-10'>{it.L('Verify Your Identity')}</h1>
+            <p>{it.L('In which country was your document issued')}</p>
+            <select className='center-text' type='select' id='residence' style={{ margin: '0 auto' , display: 'block' }} />
```

Comment:
Imho, I think it could be better to add these styles to our scss and refer to the element by id. It is harder to find and modify inline styles in general.